itou/utils/enums_context_processors.py

```diff
@@ -0,0 +1,14 @@
+import itou.job_applications.enums as job_aplication_enums
+import itou.users.enums as users_enums
+
+
+def expose_enums(request):
```

Comment:
2 en 1 : évite de passer None dans les tests, indique qu’on n’utilise pas la requête.
```suggestion
def expose_enums(*args):
```